src/containers/Accounts/components/FormModal/index.js

```diff
@@ -23,16 +23,25 @@ const FormModal = ({
   userId,
   isAuth0Enabled,
   onResetUserPassword,
+  allUserRoles,
+  extraFields,
 }) => {
   const [form] = Form.useForm();
+  const [checked, setChecked] = useState(userRole.includes('ManagePortalUser'));
```

Comment:
everything related to ManagePortalUser should not be in here. It can all be done in CMAP